README.md

```diff
@@ -25,9 +23,11 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
While the tests are running on CJ and the code below is sleeping in a loop, the users pay github runner minutes, i guess?
One day we could maybe off self-hosted runners that would execute the playwright code directly. Just a thought.